src/components/UI/TabButtons/index.js

```diff
@@ -30,7 +30,7 @@ export default ({ buttons, titles, values, selectedButton, useEqualWidthTabs, on
 
         return (
           <div
-            key={buttonName}
+            key={buttonName + index}
```

Comment:
Nice catch, thanks; when I had the inspector open I was using the three-letter week and after switching to one-letter week I didn’t check the console again so that’s probably why I missed this

Reply:
No worries, it's easy to do - for you, there was a predefined component which already uses keys, so you even had good reason to be happy with a quick glance(;